rust/operator-binary/src/restart_controller/statefulset.rs

```diff
@@ -85,30 +84,29 @@ pub async fn start(client: &Client) {
         stream::select(
             stream::select(
                 trigger_all(
-                    try_flatten_touched(
-                        reflector(cm_store, watcher(cms, ListParams::default())).inspect(|_| {
-                            cms_inited.store(true, std::sync::atomic::Ordering::SeqCst)
-                        }),
-                    ),
+                    reflector(cm_store, watcher(cms, ListParams::default()))
+                        .inspect(|_| cms_inited.store(true, std::sync::atomic::Ordering::SeqCst))
+                        .applied_objects(),
                     sts_store.as_reader(),
                 ),
                 trigger_all(
-                    try_flatten_touched(
-                        reflector(secret_store, watcher(secrets, ListParams::default())).inspect(
-                            |_| secrets_inited.store(true, std::sync::atomic::Ordering::SeqCst),
-                        ),
-                    ),
+                    reflector(secret_store, watcher(secrets, ListParams::default()))
+                        .inspect(|_| {
+                            secrets_inited.store(true, std::sync::atomic::Ordering::SeqCst)
+                        })
+                        .applied_objects(),
```

Comment:
And here.

Reply:
thx! Fixed in 9e14f9075c94887dd8151e219e3c0443010961ef